docs/src/main/paradox/docs/delta/api/views/elasticsearch-view-api.md

```diff
@@ -410,3 +410,33 @@ Request
 
 Response
 :   @@snip [restart.json](../assets/views/elasticsearch/restart.json)
+
+## Server Sent Events
+
+The server sent events response contains a series of events, represented in the following way
+
+```
+data:{payload}
+event:{type}
+id:{id}
+```
+
+where...
+
+- `{payload}`: Json - is the actual payload of the current event
+- `{type}`: String - is a type identifier for the current event. Possible types are related to core resource types (Resouce, Schema, Resolver) and available plugin types
+- `{id}`: String - is the identifier of the resource event. It can be used in the `Last-Event-Id` query parameter
+
+### Fetch failures
```

Comment:
These changes need to go in the general view page where there are the common operations for views

Fetch indexing failures

Also we need a small description.